StuffKartProject/Controllers/CheckSecurityAnswerController.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using StuffKartProject.Constant;
+using StuffKartProject.Models;
+using StuffKartProject.Services.Interfaces;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace StuffKartProject.Controllers
+{
+  [Route("ForgetPassword")]
```

Comment:
route should be ForgotPassword.
name of the controller and action method should be ForgotPasswordController and ResetPassWordController respectively.
Naming does not seem appropriate in lots of places, take a look around.

Reply:
Changed controller  name to ForgetPasswordController